src/main/java/com/github/shiranr/scooters/api/Scooters.java

```diff
@@ -26,7 +36,13 @@ public HttpResponseMessage scooters(
                     authLevel = AuthorizationLevel.ANONYMOUS)
                     HttpRequestMessage<Optional<String>> request,
             final ExecutionContext context) {
-        //TODO (shiranr) implement - committing only stubs. Next iteration will include implementation.
-        return request.createResponseBuilder(HttpStatus.OK).body("Hello").build();
+        Scooter[] scooters = service.GetAllScooters();
+        String scootersString = "";
+        try {
+            scootersString = new ScooterMixin().scooterMixin(scooters);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
```

Comment:
When run in the server, who can see this error?

Reply:
I prefer the error will not be externalized to the user and the user will get an empty list instead of exception.
I did add 500 instead of returning an empty response with 200.